parent-oss/pom.xml

```diff
@@ -54,6 +54,7 @@ governing permissions and limitations under the License.
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- this is used for both enforcing the minimum java version and as compile target -->
     <project.java.version>1.8</project.java.version>
+    <maven.plugin.version>3.6.2</maven.plugin.version>
```

Comment:
We need to wait for 3.6.4 due to https://issues.apache.org/jira/browse/MPLUGIN-384

Reply:
What is the relation of the maven.plugin version and the minimal maven version required for the resulting maven plugin? Or in other words if we use 3.6.2 of maven plugin plugin for the aemanalyser plugin, do we need to require 3.6.2 for the aemanalyser plugin?

Reply:
That said , I wouldn't mind requiring 3.6.3

Reply:
You have to distinguish between maven-plugin-api (a run time requirement has an impact on the minimum Maven version required) and Maven Plugin Tools (https://maven.apache.org/plugin-tools/index.html, only relevant at build time). This PR only upgrades the plugin build tools without an impact on the minimum Maven requirement when aemanalyser is used.

Reply:
Sure, understood - but this plugin does something which ends up in the plugin. If you are sure that this has no maven version dependency, then ok

Reply:
Yes, it has no impact on the run time. The plugin descriptor generated is still the same version as always: https://maven.apache.org/ref/3.8.4/maven-plugin-api/plugin.html